src/search.cpp

```diff
@@ -1333,6 +1346,7 @@ namespace {
           }
       }
 
+      // If move is worse than some previously searched move remember it to update it stats later
```

Comment:
If move is worse than some previously searched move remember it to update its stats later

Reply:
Actually, a comma and a _the_ is missing, too.

Should be: _If the move is worse than some previously searched move, remember it to update its stats later_